client/src/client-only-routes/ShowProjectLinks.js

```diff
@@ -173,23 +179,22 @@ const ShowProjectLinks = props => {
           solution={solution}
         />
       ) : null}
-      If you suspect that any of these projects violate the{' '}
-      <a
-        href='https://www.freecodecamp.org/news/academic-honesty-policy/'
-        target='_blank'
-        rel='noreferrer'
-      >
-        academic honesty policy
-      </a>
-      , please{' '}
-      <a
-        href={`/user/${username}/report-user`}
-        target='_blank'
-        rel='noreferrer'
-      >
-        report this to our team
-      </a>
-      .
+      <Trans i18nKey='certification.project.footnote'>
+        <a
+          href='https://www.freecodecamp.org/news/academic-honesty-policy/'
+          target='_blank'
+          rel='noreferrer'
+        >
+          academic honesty policy
+        </a>
+        <a
+          href={`/user/${username}/report-user`}
+          target='_blank'
+          rel='noreferrer'
+        >
+          report this to our team
```

Comment:
```suggestion
          report this to our team.
```
Final nitpick

Reply:
I've added the period after the closing `a` tag if that's okay because that's how it was before and in the `translations.json` 😛
```json
"footnote": "If you suspect that any of these projects violate the <0>academic honesty policy</0>, please <1>report this to our team</1>."
```

```js
    rel='noreferrer'
  >
    report this to our team
  </a>
  .
</Trans>
```